packages/core/src/state-management/__tests__/repository.test.ts

```diff
@@ -18,166 +25,219 @@ const STRING_MAP_SCHEMA = {
   },
 }
 
-describe('repository tests', () => {
-  let ipfs: IpfsApi
-  let ceramic: Ceramic
-  let repository: Repository
+let ipfs: IpfsApi
+let ceramic: Ceramic
+let repository: Repository
 
-  beforeAll(async () => {
-    ipfs = await createIPFS()
-  })
+beforeAll(async () => {
+  ipfs = await createIPFS()
+})
 
-  afterAll(async () => {
-    await ipfs.stop()
-  })
+afterAll(async () => {
+  await ipfs.stop()
+})
+
+beforeEach(async () => {
+  ceramic = await createCeramic(ipfs)
+  repository = ceramic.repository
+})
+
+afterEach(async () => {
+  jest.resetAllMocks()
+  await ceramic.close()
+})
 
-  beforeEach(async () => {
-    ceramic = await createCeramic(ipfs)
-    repository = ceramic.repository
+describe('#load', () => {
```

Comment:
it's hard to read the diff on this file.  Is it all formatting changes or are there real changes to the test logic?

Reply:
1. Remove single useless `describe` statement. Thus, formatting changes.
2. Add `sync: SYNC_ALWAYS` tests.

Existing tests are the same.